go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/clusterpedia-io/clusterpedia
 
-go 1.19
+go 1.21
```

Comment:
The version in go.mod represents the minimum version number, so I don't think we need to change it if we don't have to.

Of course we would normally update it based on the k8s dependencies

Reply:
Oh but golang 1.19 isn't supported now, Can we shift it to 1.20? As minimum supported version

Reply:
I don't think it's necessary, the minimum version is more of a minimum version dependency in terms of functionality rather than a recommended version for users. We can use the correct version at build and test time

Reply:
Understood